psh/tools/psh.py

```diff
@@ -154,10 +173,7 @@ def ls(pexpect_proc, dir=''):
     File = namedtuple('File', ['name', 'owner', 'is_dir', 'date'])
     Date = namedtuple('Date', ['month', 'mday', 'time'])
 
-    _send(pexpect_proc, f'ls -la {dir}')
-
-    if dir:
-        pexpect_proc.expect_exact(f' {dir}')
+    _send(pexpect_proc, f'ls -la {dir}') if dir else _send(pexpect_proc, f'ls -la')
 
     pexpect_proc.expect_exact('\n')
```

Comment:
Is this needed? Can it be done in _send?

Reply:
I've used strip on `cmd`, so this condition is not needed anymore :)